vernac/vernacentries.ml

```diff
@@ -1768,22 +1768,51 @@ let interp_search_restriction = function
 
 open Search
 
-let interp_search_about_item env sigma =
+let kind_searcher = Decls.(function
```

Comment:
Nit too, it seems to me that `Search` is growing pretty big here, so maybe adding `ComSearch` would make sense at this point; or even we could use `Search` itself and avoid some exports [I didn't try myself]

Reply:
We could indeed move the whole `Vernacexpr.search_item` to `Search.glob_search_item` interpretation to `search.ml`...

Reply:
As you prefer @herbelin , this is just a minor suggestion in case you think I'd be nicer to go this way.

Reply:
I finally created `comSearch.ml` and cleaned in passing the treatment of pstate/sigma in `search.ml`.

@Zimmi48: I thus pushed extra commits to the branch.